conpot/core/virtual_fs.py

```diff
@@ -15,47 +15,55 @@
 # Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-import conpot
-import os
-import sys
+import logging
+
 from fs import mountfs, tempfs, osfs
+from conpot.core.file_io import AbstractFS
+
+logger = logging.getLogger(__name__)
 
 
-class ConpotFS(object):
+# TODO: clean-up --
+class VirtualFS(object):
     """
     Conpot's virtual file system. Based on Pyfilesystem2, it would allow us to have
     arbitrary file uploads while sandboxing them - for later analysis. This is how it should look like:
 
-                          [vfs]
+                      [_conpot_vfs]
                             |
-                            |-- data (persistent)
+                            |-- data_fs (persistent)
                             |    |-- ftp/uploads
                             |    `-- misc.
                             |
-                            `-- protocols(temporary, refreshed at startup)
+                            `-- protocol_fs (temporary, refreshed at startup)
                                  |-- common
                                  |-- telnet
                                  |-- http
                                  |-- snmp
                                  `-- ftp etc.
     """
     def __init__(self):
-        self.data_fs = osfs.OSFS(root_path='')
+        self.data_fs = osfs.OSFS(root_path='')  # Specify the place where you would place the uploads
```

Comment:
`pyfilesystem2` is pretty cool. You can specify something like `ftp://will:daffodil@ftp.example.org/private` to store the uploads directly to a FTP server. 
You can even do this - `zip://projects.zip` or `tar://projects.tar` to store directly to a zip/tar file. I haven't tested these URLs though.

***Question***: What should we keep the deafult storage URL here? Since we are creating an instance of this VirtualFS class in `conpot.core.__init__.py`, I was not sure about the default file, directory - could use some advice here ;-)

Reply:
You mean where the default directory for the VFS will reside? 
$cwd/vfs seems appropriate, ... or do I miss an obvious tripwire?

Reply:
We would need to push this _default directory_ to conpot repo. Would a `conpot_vfs.tar` file [here](https://github.com/mushorg/conpot/tree/master/conpot) be okay with you?